R/densityClust.R

```diff
@@ -215,15 +219,25 @@ estimateDc <- function(distance, neighborRateLow=0.01, neighborRateHigh=0.02) {
 #' 
 #' @export
 #' 
-densityClust <- function(distance, dc, gaussian=FALSE) {
-    if(missing(dc)) {
-        dc <- estimateDc(distance)
-    }
-    rho <- localDensity(distance, dc, gaussian=gaussian)
-    delta <- distanceToPeak(distance, rho)
-    res <- list(rho=rho, delta=delta, distance=distance, dc=dc, threshold=c(rho=NA, delta=NA), peaks=NA, clusters=NA, halo=NA)
-    class(res) <- 'densityCluster'
-    res
+densityClust <- function(distance, dc, gaussian=FALSE, verbose = F) {
+  if(missing(dc)) {
+    if(verbose)
```

Comment:
The style of this package is to have braces around all statements if they start on a new line